numpy/ma/core.pyi

```diff
@@ -24,6 +25,7 @@ from numpy._globals import _NoValueType
 from numpy._typing import (
     ArrayLike,
     NDArray,
+    _ArrayLikeInt_co,
```

Comment:
Since passing booleans for `kth` is deprecated, it might be a bit cleaner to use `_ArrayLikeInt`